### ISSUE TITLE
[ISSUE]: GitReleaseManager GithubAction not logging to job

### ISSUE THREAD

**Issue by Stepami** [GitTools/actions#1224]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched [issues](https://github.com/GitTools/actions/issues) to ensure it has not already been reported

### GitReleaseManager package

GitHub Actions

### What are you seeing?

https://github.com/Stepami/hydrascript/actions/runs/10360139446/job/28678471915

In GitHub Actions logs i see no logs of tool.

But if i run it locally logs appear:
![image](https://github.com/user-attachments/assets/83980963-20ff-4e68-93d9-580895720355)
![image](https://github.com/user-attachments/assets/a8efc4dc-2282-436f-bd63-b65a896a35ea)

### What is expected?

logging to GitHub Actions Console

### Steps to Reproduce

```yml
name: Release Workflow

on:
  push:
    branches:
      - 'release'
      
jobs:
  create-release:
    name: Create release
    runs-on: ubuntu-latest
    outputs: 
      determined_version: ${{ steps.version_step.outputs.majorMinorPatch }}
    steps:
      - name: Checkout
        uses: actions/checkout@v4
        with:
          fetch-depth: 0
      - name: Setup GitVersion
        uses: gittools/actions/gitversion/setup@v3.0.0
        with:
          versionSpec: '5.12.0'
      - name: Determine Version
        id: version_step
        uses: gittools/actions/gitversion/execute@v3.0.0
        with:
          useConfigFile: true
      - name: Setup GitReleaseManager
        uses: gittools/actions/gitreleasemanager/setup@v3.0.0
        with:
          versionSpec: '0.18.x'
      - name: Create release with GitReleaseManager
        uses: gittools/actions/gitreleasemanager/create@v3.0.0
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          owner: 'Stepami'
          repository: 'hydrascript'
          milestone: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
          name: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
```

**Comment by arturcic** [comment]
@gep13

**Comment by arturcic** [comment]
> ### Prerequisites
> * [x]  I have written a descriptive issue title
> * [x]  I have searched [issues](https://github.com/GitTools/actions/issues) to ensure it has not already been reported
> 
> ### GitReleaseManager package
> GitHub Actions
> 
> ### What are you seeing?
> https://github.com/Stepami/hydrascript/actions/runs/10360139446/job/28678471915
> 
> In GitHub Actions logs i see no logs of tool.
> 
> But if i run it locally logs appear: ![image](https://private-user-images.githubusercontent.com/36167798/357219810-83980963-20ff-4e68-93d9-580895720355.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MTIyNDEsIm5iZiI6MTcyMzYxMTk0MSwicGF0aCI6Ii8zNjE2Nzc5OC8zNTcyMTk4MTAtODM5ODA5NjMtMjBmZi00ZTY4LTkzZDktNTgwODk1NzIwMzU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDA1MDU0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3OGZmNzBkNjhlMTAyYWU1NGNlYjg4YTk2NmRlOTEzMmIxN2Q0ZWNmNWY0OTc2NTJkYzU4ZGY5NDQxOWZhZWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fWK4-qHN4XKeHJYmt6V3gf6kU96XmT_p2WS_Y2eJ5cU) ![image](https://private-user-images.githubusercontent.com/36167798/357219841-a8efc4dc-2282-436f-bd63-b65a896a35ea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MTIyNDEsIm5iZiI6MTcyMzYxMTk0MSwicGF0aCI6Ii8zNjE2Nzc5OC8zNTcyMTk4NDEtYThlZmM0ZGMtMjI4Mi00MzZmLWJkNjMtYjY1YTg5NmEzNWVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDA1MDU0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4MGUxODI5YTBiY2EzOTRlNmMzYTUxOWVmODc0NGRlNWU1MWNiOWJlZGRhMjJmYzAxOTJmYTJhNTk1NmRjODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.yOF2lq-5cXBjr7wEn8z3FhDX3CaIX8ibShIfQO57vQA)
> 
> ### What is expected?
> logging to GitHub Actions Console
> 
> ### Steps to Reproduce
> ```yaml
> name: Release Workflow
> 
> on:
>   push:
>     branches:
>       - 'release'
>       
> jobs:
>   create-release:
>     name: Create release
>     runs-on: ubuntu-latest
>     outputs: 
>       determined_version: ${{ steps.version_step.outputs.majorMinorPatch }}
>     steps:
>       - name: Checkout
>         uses: actions/checkout@v4
>         with:
>           fetch-depth: 0
>       - name: Setup GitVersion
>         uses: gittools/actions/gitversion/setup@v3.0.0
>         with:
>           versionSpec: '5.12.0'
>       - name: Determine Version
>         id: version_step
>         uses: gittools/actions/gitversion/execute@v3.0.0
>         with:
>           useConfigFile: true
>       - name: Setup GitReleaseManager
>         uses: gittools/actions/gitreleasemanager/setup@v3.0.0
>         with:
>           versionSpec: '0.18.x'
>       - name: Create release with GitReleaseManager
>         uses: gittools/actions/gitreleasemanager/create@v3.0.0
>         with:
>           token: ${{ secrets.GITHUB_TOKEN }}
>           owner: 'Stepami'
>           repository: 'hydrascript'
>           milestone: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
>           name: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
> ```
> 
> ### Output log or link to your CI build (if appropriate).
> _No response_

@Stepami I hope you have revoked/removed that token you forgot to hide in the screenshot.

As for the output, at this point the GitVersion and GitReleaseManager actions/tasks are some opinionated wrappers around the command line tools, and the implementation is not logging the successful output, but only the error message if any.

I think I can agree it can be improved with a common setting to enable the tool output for all actions we support.

@Stepami are you willing to create a PR for this, and I can give some input on this.

**Comment by Stepami** [comment]
@arturcic The token is no longer available, so don't you worry)

the thing is implementation is not logging the error message as well

that is the reason i created the issue in a first place

i tried to make an automated release, but it didn't happen.
after local launch it turned out i've used labels in wrong way, which became clear from error messages i got locally on my machine

so, you should check error output firstly, i think

**Comment by siruschavoshi** [comment]
> ### Prerequisites
> 
> - [X] I have written a descriptive issue title
> - [X] I have searched [issues](https://github.com/GitTools/actions/issues) to ensure it has not already been reported
> 
> ### GitReleaseManager package
> 
> GitHub Actions
> 
> ### What are you seeing?
> 
> https://github.com/Stepami/hydrascript/actions/runs/10360139446/job/28678471915
> 
> In GitHub Actions logs i see no logs of tool.
> 
> But if i run it locally logs appear:
> ![image](https://github.com/user-attachments/assets/83980963-20ff-4e68-93d9-580895720355)
> ![image](https://github.com/user-attachments/assets/a8efc4dc-2282-436f-bd63-b65a896a35ea)
> 
> 
> ### What is expected?
> 
> logging to GitHub Actions Console
> 
> ### Steps to Reproduce
> 
> ```yml
> name: Release Workflow
> 
> on:
>   push:
>     branches:
>       - 'release'
>       
> jobs:
>   create-release:
>     name: Create release
>     runs-on: ubuntu-latest
>     outputs: 
>       determined_version: ${{ steps.version_step.outputs.majorMinorPatch }}
>     steps:
>       - name: Checkout
>         uses: actions/checkout@v4
>         with:
>           fetch-depth: 0
>       - name: Setup GitVersion
>         uses: gittools/actions/gitversion/setup@v3.0.0
>         with:
>           versionSpec: '5.12.0'
>       - name: Determine Version
>         id: version_step
>         uses: gittools/actions/gitversion/execute@v3.0.0
>         with:
>           useConfigFile: true
>       - name: Setup GitReleaseManager
>         uses: gittools/actions/gitreleasemanager/setup@v3.0.0
>         with:
>           versionSpec: '0.18.x'
>       - name: Create release with GitReleaseManager
>         uses: gittools/actions/gitreleasemanager/create@v3.0.0
>         with:
>           token: ${{ secrets.GITHUB_TOKEN }}
>           owner: 'Stepami'
>           repository: 'hydrascript'
>           milestone: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
>           name: 'v${{ steps.version_step.outputs.majorMinorPatch }}'
> ```
> 
> ### Output log or link to your CI build (if appropriate).
> 
> _No response_

**Comment by arturcic** [comment]
Fixed by #1341
